### ISSUE TITLE
App crashes when moving songs in queue

### ISSUE THREAD

**Issue by DarkblooM-IO** [fast4x/RiMusic#3460]
### Steps to reproduce the bug

1. Launch a queue (playlist, radio, ...)
2. Open song queue (unlock it if necessary)
3. Try moving any song

### Expected behavior

App should not crash and song should be placed in queue accordingly

### Actual behavior

App crashes

### Logs

```
2024-08-28T16:59:00.488:    java.lang.NoSuchMethodError: No virtual method removeFirst()Ljava/lang/Object; in class Ljava/util/ArrayList; or its super classes (declaration of 'java.util.ArrayList' appears in /apex/com.android.art/javalib/core-oj.jar)
     V5.a.a(SourceFile:1) 
     A1.d.b(SourceFile:90) 
     V5.i.u(SourceFile:101) 
     g7.a.i(SourceFile:9) 
     z7.J.run(SourceFile:113) 
     J0.d0.h0(SourceFile:7) 
     J0.c0.doFrame(SourceFile:10) 
     android.view.Choreographer$CallbackRecord.run(Choreographer.java:1548) 
     android.view.Choreographer$CallbackRecord.run(Choreographer.java:1559) 
     android.view.Choreographer.doCallbacks(Choreographer.java:1118) 
     android.view.Choreographer.doFrame(Choreographer.java:1004) 
     android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1526) 
     android.os.Handler.handleCallback(Handler.java:958) 
     android.os.Handler.dispatchMessage(Handler.java:99) 
     android.os.Looper.loopOnce(Looper.java:232) 
     android.os.Looper.loop(Looper.java:334) 
     android.app.ActivityThread.main(ActivityThread.java:8293) 
     java.lang.reflect.Method.invoke(Native Method) 
     com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592) 
     com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1062)
```

### RiMusic version

0.6.49.2

### Android version

Android 14

**Comment by fast4x** [comment]
Just fixed and available from version 0.6.50

**Comment by DarkblooM-IO** [comment]
> Just fixed and available from version 0.6.50

Neat
